### PR TITLE
feat/Create utils to format runtime and currency outputs

### PR DESCRIPTION
**Pull request by trqhien**
#### What does this PR do?
- Convert raw outputs from API to user-friendly value

#### What are the relevant issues?
- https://github.com/trqhien/progressive-movie-web-app/issues/15

### CODE REVIEW

File: public/api.js [hunk 1]
```diff
@@ -6,6 +6,43 @@ function asJSON(response) {
   return response.json();
 }
 
+function convertToRuntimeFormat(durationInMinutes) {
+  if (typeof durationInMinutes !== 'number') {
+    return '';
+  }
+
+  const hours = Math.floor(durationInMinutes / 60);
+  const minutes = durationInMinutes - hours * 60;
```
Comment (khiemle) [inline public/api.js]: Does it have `%` operator?

File: public/api.js [hunk 2]
```diff
@@ -15,13 +52,13 @@ function updateUI(movie) {
   overview.innerHTML = movie.overview;
 
   const runtime = document.getElementById('runtime-label');
-  runtime.innerHTML = movie.runtime;
+  runtime.innerHTML = `${convertToRuntimeFormat(movie.runtime)}`;
 
   const budget = document.getElementById('budget-label');
-  budget.innerHTML = movie.budget;
+  budget.innerHTML = `$${formatCurrency(movie.budget)}`;
```
Comment (khiemle) [inline public/api.js]: It's having double `$`

File: public/api.js [hunk 3]
```diff
@@ -15,13 +52,13 @@ function updateUI(movie) {
...
 
   const revenue = document.getElementById('revenue-label');
-  revenue.innerHTML = movie.revenue;
+  revenue.innerHTML = `$${formatCurrency(movie.revenue)}`;
```
Comment (khiemle) [inline public/api.js]: It's having double $

File: public/api.js [hunk 4]
```diff
@@ -6,6 +6,43 @@ function asJSON(response) {
...
+      style: 'decimal',
+      maximumFractionDigits: 2,
+      minimumFractionDigits: 2,
+    },
+  );
+
+  return currencyFormatter.format(num);
+}
+
 function updateUI(movie) {
```
Comment (khiemle) [inline public/api.js]: It looks be ok but i think,We should format the value when bind the `movie` model to view

File: public/api.js (hunk 2 above)
Comment (trqhien) [reply public/api.js]: that's exactly how I want it.
image

File: public/api.js (hunk 1 above)
Comment (trqhien) [reply public/api.js]: actually it has. 👍

File: public/api.js (hunk 2 above)
Comment (khiemle) [reply public/api.js]: oh, i see 😆

File: public/api.js (hunk 3 above)
Comment (trqhien) [reply public/api.js]: this's exactly what I want. The first `$` is for the actual currency unit. So it will be displayed as `$260,000,000`

**Pull request merged**